index.d.ts

```diff
@@ -33,8 +33,10 @@ declare module "replicate" {
       created_by: Account;
       configuration: {
         hardware: string;
-        min_instances: number;
-        max_instances: number;
+        scaling: {
+          min_instances: number;
+          max_instances: number;
+        };
```

Comment:
This was incorrect in the original PR adding `deployments.get` #206

Reply:
Was I out of touch? No, it was the OpenAPI specification that was wrong https://github.com/replicate/replicate-javascript/pull/232